AssetHelper/CatalogTools/Catalog.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.AddressableAssets.ResourceLocators;
+using UnityEngine.ResourceManagement.ResourceLocations;
+using UnityEngine.ResourceManagement.Util;
+
+namespace Silksong.AssetHelper.CatalogTools
+{
+
+    /// <summary>
+    /// Class to write a catalog file that can be loaded by the Addressables package.
+    /// </summary>
+    internal static class Catalog
```

Comment:
Same here, I think this should be called CatalogHelper, CatalogFactory, CatalogUtil(s) or something like that (to indicate it's used to make a catalog, rather than an instance represents a catalog